docs/pages/documentation/11-examples/example-twitter-streaming-public-tweets.md

```diff
@@ -0,0 +1,580 @@
+# Using Grakn To Work With Tweet Data
+In this tutorial we will look at how to stream public tweets into Grakn's knowledge graph. The tutorial aims to demonstrate key concepts such as receiving, inserting and querying data. Upon the completion of this tutorial, you will have learnt about these concepts:
+
+- Defining a simple Grakn.ai ontology using the Java API
+- Streaming public tweets into the application with the [Twitter4J](http://twitter4j.org/ "Twitter4J") library
+- Inserting tweets into the knowledge graph using Grakn's Graph API
+- Performing simple queries using GraQL, the Grakn's Query Language
+
+## Registering Your Own Twitter Application
+As of today, you will need a valid credential in order to call practically every endpoint in the Twitter API. Therefore, you must own a Twitter application (or, register a new one) before proceeding further.
+
+You can register your own application in the [Twitter Application Management](https://apps.twitter.com/). Once you've done so you can  get the credentials by visiting the **Keys and Access Tokens** tab. The value we care about in particular are Consumer Key, Consumer Secret, Access Token, and Access Token Secret.
+
+## Bootstraping The Skeleton Project
+Let's bootstrap a new maven project! Hit the command line and run the following command to generate a new maven project:
+
+```sh
+mvn archetype:generate -DgroupId=ai.grakn -DartifactId=twitterexample -DarchetypeArtifactId=maven-archetype-quickstart -DinteractiveMode=false
+```
+
+Now that you have basic project structure and `pom.xml` in place, let's start customising them to our needs. We will add two things to the `<build>` section:
+
+1. `maven-compiler-plugin` configuration in order to enable lambda and other nifty Java 8 features
+2. `maven-shade-plugin` configuration which points to our `Main` class, for creating a fat JAR.
```

Comment:
Ah yes. You right. I do not have a better suggestion actually. So I suppose this can stay.